test/puppeteer.test.js

```diff
@@ -75,7 +77,7 @@ describe('Apify.launchPuppeteer()', () => {
         expect(Apify.launchPuppeteer({ proxyUrl: 'socks4://user:pass@example.com:1234' })).rejects.toThrow(Error);
         expect(Apify.launchPuppeteer({ proxyUrl: 'socks5://user:pass@example.com:1234' })).rejects.toThrow(Error);
         expect(Apify.launchPuppeteer({ proxyUrl: ' something really bad' })).rejects.toThrow(Error);
-        expect(Apify.launchPuppeteer({ proxyUrl: 'xxx', useApifyProxy: true })).rejects.toThrow(Error);
+        expect(Apify.launchPuppeteer({ proxyUrl: 'xxx', proxyConfiguration })).rejects.toThrow(Error);
```

Comment:
The `proxyConfiguration` shouldn't be here and it should'nt have any effect. Does it throw?

Reply:
Nope, forgot to remove it after we decided to not use `proxyConfiguration` there.